ui/src/components/Rules/RuleGroup/AddModal/index.tsx

```diff
@@ -441,627 +432,671 @@ const AddModal = (props: AddModal) => {
             is required
           </Alert>
         ) : undefined}
-        <div className="section">
-          <form>
-            <div className="form-row">
-              <label htmlFor="name" className="text-label">
-                Name *
-              </label>
-              <div className="form-input">
-                <div className="form-input-field">
-                  <input
-                    name="name"
-                    id="name"
-                    type="text"
-                    ref={nameRef}
-                    defaultValue={props.editData?.name}
-                  ></input>
-                </div>
-              </div>
-            </div>
-
-            <div className="form-row">
-              <label htmlFor="description" className="text-label">
-                Description
-              </label>
-              <div className="form-input">
-                <div className="form-input-field">
-                  <textarea
-                    name="description"
-                    id="description"
-                    rows={5}
-                    defaultValue={props.editData?.description}
-                    ref={descriptionRef}
-                  ></textarea>
-                </div>
-              </div>
-            </div>
+        <form className="flex flex-col">
+          <div className="grid grid-cols-1 gap-x-5 gap-y-3 md:grid-cols-2">
```

Comment:
I think the breakpoints need some tweaking. I personally think that `md:grid-cols-2` should be `xl:grid-cols-2` or even `2xl:grid-cols-2` as most input labels are wrapping after 1 or 2 words. I think leaving all child md breakpoints as-is is fine, it gives the larger screens a less compressed view.

```suggestion
          <div className="grid grid-cols-1 gap-x-5 gap-y-3 xl:grid-cols-2">
```

Reply:
The grid/flex layout is a bit wonky, is there negative margins somewhere? Options also has quite a bit of space on the right that could be used up on the smaller screens.

![image](https://github.com/user-attachments/assets/06386408-e32c-4c80-90db-e6d22734564a)

Reply:
I agree with your statements, after review. I am not completely sold on 2xl:grid-cols-2 but I did change it to xl. I also made some other changes that I think you may like better. Let me know what you think.